todo-app/src/Task.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import {inject, observer} from 'mobx-react';
+import TaskBody from './TaskBody';
+import './index.css';
+
+@inject('taskStore') 
+@observer
+export default class Task extends React.Component {
+
+
+  render() {
+      const{todoItem, toggleStatus, isEditing, onRemove, onEdit, onCancleEdit, onSaveEdit} = this.props;
+        return (
+            <li className="task-list">
+             <span> <input type = 'checkbox' checked = {todoItem.done} onChange = {() => toggleStatus(todoItem.id)}/></span>
```

Comment:
I cannot even read this file due to formatting. Please reformat with proper HTML indentation.